### PR TITLE
Add optional negatives to unique dates in level 1 and level 2.

### PR DESCRIPTION
**Pull request by somexpert**
ping @ldko, @madhulika95b

### CODE REVIEW

**Comment by somexpert** [comment]
This will resolve #15, or at least in the sense of resolving it against the older version of the spec

File: tests/test_valid_edtf.py [hunk 1]
```diff
@@ -168,9 +177,11 @@ class TestLevel1(object):
         '1984-06-02?/2004-08-08~',
         '1984-06-02?/unknown',
         'y170000002',
+        'y170000002',
```
Comment (somexpert) [inline tests/test_valid_edtf.py]: this was a mistake, I'll get rid of it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ Change Log
 x.x.x
 =====
 
+* Fixed an issue where some negative dates weren't validating under level 1 or level 2.
```
Comment (ldko) [inline CHANGELOG.md]: You could add a link to the related issue.

Comment (somexpert) [reply CHANGELOG.md]: Done.

**Pull request merged**